jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -197,10 +198,14 @@ jobs:
       - run: bundle install --path vendor/bundle
 
       # The DB is not automatically created in 2.0, so we manually set up a test DB
-      - run: sudo -u root createuser -h localhost --superuser ubuntu
-      - run: sudo createdb -h localhost test_db
+      # If your project is Rails, then 'db:create' should create a test DB for you.
       - run: bundle exec rake db:create db:schema:load
 
+      # You can also create a test DB manually if your project requires.
+      # We are installing postgresql package because we need 'createdb' command
+      - run: apt-get update -qq; apt-get install -y postgresql
```

Comment:
@kimh it would be better to specify a primary container image that comes with Postgres client. `apt-get update` isn't ideal in 2.0 docs for several reasons:

- it's slow
- it's ubuntu / debian specific
- it's mutable since apt repositories change. we are aiming for immutability to make things more reliable

we should move away from how we did things in 1.0.